src/node_report.cc

```diff
@@ -436,20 +446,57 @@ static void PrintNetworkInterfaceInfo(JSONWriter* writer) {
   }
 }
 
+static void PrintJavaScriptErrorProperties(JSONWriter* writer,
+                                           Isolate* isolate,
+                                           Local<Object> error) {
+  writer->json_objectstart("errorProperties");
+  if (!error.IsEmpty()) {
```

Comment:
```suggestion
  if (!error.IsEmpty()) {
  TryCatch try_catch(isolate);
```

Otherwise this may leave exceptions from below lying around. This silences the error altogether but I don’t really see what else we could do. (Ditto below).